src/OrchardCore/OrchardCore.Data.YesSql.Abstractions/IScopedIndexProvider.cs

```diff
@@ -2,11 +2,14 @@
 
 namespace OrchardCore.Data
 {
+
+#pragma warning disable CS1574 // XML comment has cref attribute that could not be resolved
```

Comment:
Why?

Reply:
![image](https://user-images.githubusercontent.com/24724371/170166966-83c430fe-0dd1-4a32-a9a1-4844243a943f.png)
I get compiling error otherwise.

Reply:
Maybe just a missing `using YesSql` and then just do a cref on ISession

Reply:
It should work too, unless `Http.Abstractions` is imported

Reply:
okay. this change fixed it.
![image](https://user-images.githubusercontent.com/24724371/170168421-008750d1-ee94-4ebb-bb16-7792195ab13e.png)
I'll commit the change next